pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -31,6 +36,23 @@ import (
 	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/util"
 )
 
+var knownControllers = []string{
```

Comment:
Should I get this from somewhere else?

Reply:
Worth defining these as constants and checking for them through constants instead of literal strings. Suitable for moving the constants next to https://github.com/openshift/api/blob/master/openshiftcontrolplane/v1/types.go#L200C6-L200C38 given they are used in https://github.com/openshift/openshift-controller-manager/blob/80c4923e0b2014ec8b8a6845c21078e6cc036f91/pkg/cmd/controller/config.go#L3-L25 as well.